content/collections/docs/collections.md

```diff
@@ -281,29 +280,29 @@ route: /{parent_uri}/{slug}.html
 ```
 
 #### Organizing sports brackets with structures
-
-Here we use the "team" field, defined in the collection's blueprint and the "depth" meta variable
+Here's how we use the `depth` variable, along with the `team_name` field from the entry's blueprint.
 
 ``` yaml
-route: /tournament/round-{depth}/{team}
+route: /tournament/round-{depth}/{team_name}
 # example: /tournament/round-4/chicago-bulls
 ```
 
-#### For when your entry depend on it's related content
+#### Using fields from related entries
+For example, if you have a `category` field in your Products collection and you'd like to your product URLs to depend on it, you can configure a [computed value](/computed-values) to return the category URL, then use that computed value in your collection's route:
 
-For instance, if you have a "category" field in your "products" collection and want your products url depend on it. 
-You can use "computed values" to grab your category url
+``` php
+// app/Providers/AppServiceProvider.php
+
+use Statamic\Facades\Collection;
 
-```php
-// define the computed value in a service provider
 Collection::computed('products', 'category_url', function ($entry, $value) {
     return $entry->category?->url();
 });
 ```
 
 ``` yaml
 route: '{{ category_url }}/{{ slug }}'
-# example: /wooden-toys/steam-locomotive
+# example: /categories/wooden-toys/steam-locomotive
```

Comment:
@duncanmcclean , actually the /categories/ prefix here does not match the pattern given in the yaml route above.
either you add it to the route defintion : 

`route: 'categories/{{ category_url }}/{{ slug }}'`

or you remove it from your example.

what do you think ?

Reply:
I added it to try and make it clearer that the `/categories/wooden-toys` part *was* the `{{ category_url }}` and you're just appending the slug to the end of it.